.github/ISSUE_TEMPLATE/user_research.yml

```diff
@@ -0,0 +1,30 @@
+name: User Research
+description: Define new user research
+title: User Research for ...
+labels: ["user testing"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to create this research story!
+
+        In most cases, the Qiskit Textbook maintainers are the only ones creating and performing user research to test new ideas. If you believe you need to create a research task, please continue and provide as much information as you can.
+  - type: textarea
+    id: background
+    attributes:
+      label: Background
+      description: Set the scene and describe the motivation to conduct this user research
+  - type: textarea
+    id: audience
+    attributes:
+      label: Audience
+      description: High-level description of the intended audience 
+    validations:
+      required: true
+  - type: textarea
+    id: relevant
+    attributes:
+      label: Relevant existing research and issues 
+      description: Is 
```

Comment:
I don't understand this description. Was is unintended? If not, could you please help me understand what it means.

Thank you! 🙂

Reply:
Sure! There are two main use cases:

- Epics: sometimes we create user stories that are massive and should have multiple user stories attached, like Assignments and Problem Sets and Problem Set Discoverability
- Existing issues: sometimes we have old issues that are consolidated and formalised into a user story, like Equation Numbering

Reply:
Resolved offline. `description: Is` can be removed.